### DISCUSSION TITLE
adding Layers from ESRI-VectorTileServer

### DISCUSSION THREAD

**Discussion by leylines** [TerriaJS/terriajs#7032] [Q&A] [answered]
I would like to add some Layers from ESRI-VectorTileServer-Services. My catalog-entry look like this:

            {
              "id": "environment-artificial-world-transportation",
              "type": "mvt",
              "name": "World Transportation",
              "description": "This map presents transportation data, including highways, roads, railroads, and airports for the world.",
              "url": "https://basemaps.arcgis.com/arcgis/rest/services/World_Basemap_v2/VectorTileServer/tile/{z}/{y}/{x}.pbf",
              "dataCustodian": "[arcgis.com](https://www.arcgis.com/home/item.html?id=30d6b8271e1849cd9c3042060001f425)"
            },

But this does not work.

Is it possible to add an ESRI-VectorTileServer-Service?

Thank you,

Jörg

**Comment by techmavengeospatial** [comment, marked as answer]
styleUrl  is missing
https://docs.terria.io/guide/connecting-to-data/catalog-type-details/mvt/
In general, Terriajs does not do complex vector tiles like basemaps with lots of layers
this is URL to stylesheet
https://basemaps.arcgis.com/arcgis/rest/services/World_Basemap_v2/VectorTileServer/resources/styles
